docs/frontend.md

```diff
@@ -1,33 +1,24 @@
-- Will use `test-`* prefixed classes for testing hooks, mainly integration testing.
-- Uses the [18f linting config](https://raw.githubusercontent.com/18F/frontend/18f-pages-staging/.scss-lint.yml) for scss linting.
-- Use `scss-lint` which runs on the latest PR commits.
-- CodeClimate will notify on a PR on Github if there are any linting infractions.
-- Always default to semantic HTML.
+# Front-end architecture
 
-## JavaScript
-- The site should work without JavaScript because there is no need for complex user interactions that would use JS.
-- The site should not require jQuery.
-- Should use the AirBnB linter configuration for JavaScript.
-- CodeClimate will notify on a PR on Github if there are any linting infractions.
-- All front end dependencies should be expressed in `package.json`.
-- Many of the dependencies for the app are in the `Gemfile` since it is a Rails app.
-- Front end dependencies will be upgraded when a vurlnability is found.
-- Just one JavaScript file will be bundled up and sent to the client
+### CSS + HTML
+- utilizes single-purposed, reusable utility classes (via `Basscss`) to build UI components
+  - utility classes should do one thing and do it well, should be simple and obvious to use, and should operate independently
+  - if the markup for something is defined multiple times across templates, it should be consolidated to a single place
+- leverages elements from the U.S. Web Design Standards (i.e., fonts, colors)
+- uses Sass as the CSS preprocessor and `scss-lint` to keep files tidy
+- uses well structured, accessible, semantic HTML
+
+### JavaScript
+- site should work if JS is off (and have enhanced features if JS is on) 
+- uses AirBnB's ESLint config
+- JS modules are installed & managed via `npm` (see `package.json`)
+- JS is transpiled, bundled, and minified via `browserify` (using `browserify-rails` gem to utilize Rails asset pipeline)
 
 ### Testing
-- Integration tests and unit tests should always be running without failure.
-- Tests should be added/updated with new functionality and when features are changed.
-- Attempt to unit test data related JS; functional/integration tests are fine for DOM related code
-- If you’re having problems with you functional testing framework being consistent (random failures, fragile tests that always break) consider not adding new tests and disabling broken tests after ~10 minutes of attempting to fix them to ensure the team can move quickly and without too much frustration.
+- integration tests and unit tests should always be running and passing
+- tests should be added/updated with new functionality and when features are changed
+- attempt to unit test data-related JS; functional/integration tests are fine for DOM-related code
 
-## Devices
-- Strive to support all browsers with > 1% usage
-- The site should work well for users across all device sizes
-- Accessibility HTML code sniffer should be hooked into build process
```

Comment:
nothing at the moment; will add an automated build step here, and update the docs once that's in place